zulipterminal/ui_tools/buttons.py

```diff
@@ -106,6 +106,18 @@ def __init__(self, controller: Any, width: int, count: int=0) -> None:
                          width=width)
 
 
+class MentionedButton(TopButton):
+    def __init__(self, controller: Any, width: int, count: int=0) -> None:
+        button_text = ("Mentions [" +
```

Comment:
Maybe we should adjust the spacing to look a little better, hard-coded for now?

Reply:
Anything particular in mind?